src/app.js

```diff
@@ -22,9 +23,9 @@ uploadForm.addEventListener("submit", (e) => {
       const text = e.target.result;
       const data = csvToArray(text);
       console.log(data);
-      preview.innerHTML = generateTable(data);
+      preview.appendChild(generateTable(data));
```

Comment:
Using `appendChild` instead of replacing the HTML creates a bug.
If you click submit multiple times the _Download CSV_ and the result table get duplicated.
<img width="1920" alt="Screenshot 2021-06-09 at 08 28 59" src="https://user-images.githubusercontent.com/1199991/121304648-e8eb9880-c8fc-11eb-9c55-20c95c7ea527.png">

Reply:
I've fixed this by amending a few things:

* There was a bug as a result of me incorrectly using the `html-webpack-plugin` template which led to app.js being sourced twice in the HTML (fixed with 
77f293b ) 
* I was using `appendChild` here and also with the download buttons. Moved to using `innerHTML` and sourcing the HTML object's `outerHTML` to rewrite the page each time you submit the files (fixed with 
ac07a0d).